app/src/main/java/org/covidwatch/android/extension/Tasks.kt

```diff
@@ -22,6 +22,16 @@ suspend fun <T> Task<T>.await(): Either<Int, T> = withContext(Dispatchers.IO) {
     }
 }
 
+suspend fun Task<Void>.awaitNoResult(): Either<ENStatus, Void?> = await().let {
```

Comment:
What is a nullable void

Reply:
Yep, doesn't make sense but I had to please nullability of the result. It's because `FakeExposureNotification` return `null` instead of null. This is the only acceptable object to return that the compiler accepts as Java's `Void`.

Thus we have a nullable void -- welcome to the world of two languages that can be written inside of each other? :)